libcaliptra/examples/generic/main.c

```diff
@@ -263,7 +275,7 @@ int rt_test_all_commands()
 {
     int failure = 0;
     uint32_t non_fatal_error;
-    int status = boot_to_ready_for_fw();
+    int status = boot_to_ready_for_fw(false);
 
     if (status){
```

Comment:
why are we overriding the actual cause of failure with 1 ?

Reply:
I don't have the context. I was not the original implementer of the testbench.